### PR TITLE
TG-63: Include shared components into projects 

### PR DESCRIPTION
**Pull request by gavafue**

### CODE REVIEW

File: src/Components/Projects/index.js [hunk 1]
```diff
@@ -1,51 +1,95 @@
-//import styles from './projects.module.css';
```
Comment (paurinaldi) [inline src/Components/Projects/index.js:1]: Gabi, maybe you could've added a stylesheet for Project's index, so you could format the contents a little more like figma's references. Also, today we agreed we would use the secondary style on Button component for submit and add.

**Review by paurinaldi — APPROVED** [review]

File: src/Components/Projects/Form/form.module.css [hunk 2]
```diff
@@ -11,42 +11,29 @@ form {
...
-  border-radius: 5px;
-}
-
 label {
   font-size: 15px;
   margin-left: 2%;
 }
+select {
+  width: 100% !important;
+}
```
Comment (samueltrilloRR) [inline src/Components/Projects/Form/form.module.css:22]: Tags don't work properly with css modules. Please, replace them with classes

File: src/Components/Projects/Form/index.js [hunk 3]
```diff
@@ -103,120 +110,121 @@ const Form = () => {
...
-        // }, 3000);
+        setInfoForFeedback({ title: 'Something went wrong!', description: data.message });
+        setShowFeedbackMessage(true);
       }
     } catch (err) {
       console.log(err);
     }
   };
+  const arrayToMapEmployees = employees.map((employee) => {
+    return { id: employee._id, optionContent: employee.firstName + employee.lastName };
```
Comment (samueltrilloRR) [inline src/Components/Projects/Form/index.js:125]: Replace oprionContent concat using ES6

File: src/Components/Projects/Form/index.js [hunk 4]
```diff
@@ -103,120 +110,121 @@ const Form = () => {
...
-        </button>
-      </form>
-      {showFeedbackModal && (
-        <FeedbackModal
-          feedbackTitle={contentFeedbackModal.title}
-          messageContent={contentFeedbackModal.description}
-          setShowFeedbackModal={setShowFeedbackModal}
-          showFeedbackModal={showFeedbackModal}
+        <label htmlFor="name">Name</label>
+        <InputText
```
Comment (samueltrilloRR) [inline src/Components/Projects/Form/index.js:133]: InputText and InputSelect components already have label tag, so label tags on this component should be deleted, and pass the label to the InputText and InputSelect components

File: src/Components/Projects/index.js [hunk 5]
```diff
@@ -1,51 +1,101 @@
...
-import ProjectsTable from './Table';
 import { useEffect, useState } from 'react';
+import Table from '../Shared/Table/index';
+import DeleteMessage from '../Shared/DeleteMessage';
+import Modal from '../Shared/Modal';
+import FeedbackMessage from '../Shared/FeedbackMessage';
+import Button from '../Shared/Button';
+import styles from './projects.module.css';
+const editData = (id) => {
+  window.location = `/projects/form/${id}`;
```
Comment (samueltrilloRR) [inline src/Components/Projects/index.js:9]: Should use React Router's hook to handle redirection

File: src/Components/Projects/index.js [hunk 6]
```diff
@@ -1,51 +1,101 @@
...
 
 const Projects = () => {
   const [projects, setProjects] = useState([]);
-  const [showModal, setShowModal] = useState(false);
-  const URL = `${process.env.REACT_APP_API_URL}/projects`;
+  const [showDeleteMessage, setShowDeleteMessage] = useState(false);
+  const [showFeedbackMessage, setShowFeedbackMessage] = useState(false);
+  const [infoForDelete, setInfoForDelete] = useState('');
+  const [infoForFeedback, setInfoForFeedback] = useState({});
+  const URL = `${process.env.REACT_APP_API_URL}`;
```
Comment (samueltrilloRR) [inline src/Components/Projects/index.js:18]: On this case, is not necessary a concatenation

File: src/Components/Projects/index.js [hunk 7]
```diff
@@ -1,51 +1,101 @@
...
-        <a href="/projects/form">Add a Project</a>
-      </button>
+    <section className={styles.container}>
+      <div className={styles.indexTableContainer}>
+        <h2>Projects</h2>
+        <div className={styles.buttonContainer}>
+          <Button
+            label="Add a Project"
+            theme="secondary"
+            onClick={() => (window.location = '/projects/form')}
```
Comment (samueltrilloRR) [inline src/Components/Projects/index.js:63]: Use React Router's hooks to handle this

**Review by samueltrilloRR — CHANGES_REQUESTED** [review]

**Review by OctavioRodriguezPaseyro — APPROVED** [review]

**Review by samueltrilloRR — APPROVED** [review]

**Pull request merged**